From 6f3535a520674b757548c1c84a48150e12cbb739 Mon Sep 17 00:00:00 2001
Message-Id: <6f3535a520674b757548c1c84a48150e12cbb739.1528633460.git.mjg@fedoraproject.org>
From: Michael J Gruber <mjg@fedoraproject.org>
Date: Sun, 10 Jun 2018 14:22:50 +0200
Subject: [PATCH] Fix 699271: skip space correctly

Terminate skipping spaces when there is no more space or no more char.
---
 source/pdf/pdf-xref.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/source/pdf/pdf-xref.c b/source/pdf/pdf-xref.c
index 8515090d..68a8783b 100644
--- a/source/pdf/pdf-xref.c
+++ b/source/pdf/pdf-xref.c
@@ -647,7 +647,7 @@ fz_skip_space(fz_context *ctx, fz_stream *stm)
 	do
 	{
 		int c = fz_peek_byte(ctx, stm);
-		if (c > 32 && c != EOF)
+		if (c > 32 || c == EOF)
 			return;
 		(void)fz_read_byte(ctx, stm);
 	}
-- 
2.18.0.rc1.265.g7dd663a68f

